src/css/style.css

```diff
@@ -1,75 +1,110 @@
 * {
-	padding: 0;
-	margin: 0;
-	box-sizing: border-box;
-	font-family: 'Roboto', sans-serif;
-	color: #2E3E48;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  color: #2e3e48;
 }
 
 .text-supporting {
-	color: #828282;
+  color: #828282;
 }
 
 .gallery {
-	background: rgba(200, 254, 200, 0.2); /*#C8FEC8*/
-	padding: 75px;
+  background: rgba(200, 254, 200, 0.2); /*#C8FEC8*/
+  padding: 75px;
 }
 
-.gallery-title {
-	margin-bottom: 45px;
-	color: #006400;
-	text-align: center;
+.involve {
+  background: #ffffff;
+  padding: 100px 0;
+  text-align: center;
+}
+
+.gallery-title,
+.involve-title {
+  margin-bottom: 45px;
+  color: #006400;
+  text-align: center;
 }
 
 .gallery-grid-container {
-	display: grid;
-	grid-template-columns: repeat(2, 1fr);
-	grid-template-rows: repeat(2, 1fr);
-	grid-gap: 5px;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  grid-gap: 5px;
 }
 
 .gallery-grid-leftCol {
-	justify-self: end;
+  justify-self: end;
 }
 
 .gallery-grid-rightCol {
-	justify-self: start;
+  justify-self: start;
 }
 
 .card {
-	border-radius: 5px;
-	box-shadow: 0 10px 18px 0 rgba(0,0,0,0.2);
-	background: white;
-	width: 435px;
-	margin: 25px;
+  border-radius: 5px;
+  box-shadow: 0 10px 18px 0 rgba(0, 0, 0, 0.2);
+  background: white;
+  width: 435px;
+  margin: 25px;
 }
 
 .card-photo {
-	border-radius: 5px 5px 0 0;
-	width: 100%;
-	height: 250px;
-	object-fit: cover;
+  border-radius: 5px 5px 0 0;
+  width: 100%;
+  height: 250px;
+  object-fit: cover;
 }
 
 .card-title {
-	font-size: 1em;
-	font-weight: bold;
-	padding: 5px 0 5px 0;
+  font-size: 1em;
+  font-weight: bold;
+  padding: 5px 0 5px 0;
 }
 
 .card-text {
-	padding: 5px 0 5px 0;
+  padding: 5px 0 5px 0;
 }
 
 .card-secondary-text {
-	font-size: 0.8em;
+  font-size: 0.8em;
 }
 
 .card-text-container {
-	padding: 10px 25px 15px 25px;
+  padding: 10px 25px 15px 25px;
 }
 
 .map-icon {
-	position: relative;
-	top: 2px;
-}
\ No newline at end of file
+  position: relative;
+  top: 2px;
+}
+
+.message {
+  width: 600px;
+  margin: 50px auto;
+  font-size: 18px;
+  line-height: 30px;
+  text-align: center;
+  color: #2e3e48;
+}
+
+.button-primary {
+  background: #006400;
+  border-radius: 5px;
+  padding: 12px 16px;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  color: #ffffff;
+  text-decoration: none;
+  transition: 300ms ease all;
+  display: inline-block;
+  box-shadow: 0 0 0 rgba(200, 254, 200, 0);
+}
+
+.button-primary:hover {
```

Comment:
Hover effect is great, but can we just do change of color to either darker or lighter of the current color? 😄

Reply:
i use 0to255.com for darker current color since it's not on project plan 😅sorry.
<img width="471" alt="CleanShot 2019-10-06 at 07 07 15@2x" src="https://user-images.githubusercontent.com/54263905/66261900-3babad80-e808-11e9-8933-3bc77c2f29e6.png">